File: Sakila_Net/Sakila_Net/VB_SakilaLib/Neg/NegPayment.vb
Imports  SakilaLib.Cl.Harrys.Sakila.DAO
Imports  SakilaLib.Cl.Harrys.Sakila.Model
Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace Neg

                Public Class ClNegPayment
                    Public Shared Function listar(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAO.DAOPayment.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public  Shared Function consultar( payment as ClPayment) as Boolean
                        Return DAO.DAOPayment.SelectOne(payment)     
                    End Function
                    Public  Shared Function crear( payment as ClPayment) as Integer
                        Return DAO.DAOPayment.SqlInsert(payment)     
                    End Function
                    Public  Shared Function eliminar( payment as ClPayment) as Integer
                        Return DAO.DAOPayment.SqlDelete(payment)     
                    End Function
                    Public  Shared Function actualizar( payment as ClPayment) as Integer
                        Return DAO.DAOPayment.SqlUpdate(payment)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  

File: Sakila_Net/Sakila_Net/CS_SakilaLib/Neg/NegPayment.cs
using System.Data;
using Cl.Harrys.Sakila.DAO;
using Cl.Harrys.Sakila.Model;
namespace Cl.Harrys.Sakila.Neg
{
    public class ClNegPayment
    {
        public static DataTable listar(int nPagina ,int cantidad)
        {
            return DAO.DAOPayment.sqlSelectAll(nPagina, cantidad);
        }
        public  static bool consultar( ClPayment payment) 
        {
            return DAO.DAOPayment.sqlSelect(payment);
        }
        public  static int crear(ClPayment payment)
        {
            return DAO.DAOPayment.sqlInsert(payment);
        }
        public  static int eliminar(ClPayment payment)
        {
            return DAO.DAOPayment.sqlDelete(payment);
        }
        public  static int actualizar(ClPayment payment)
        {
            return DAO.DAOPayment.sqlUpdate(payment);
        }
    }
}
